src/llm/apis/openai_completions.hpp

```diff
@@ -158,7 +157,7 @@ class OpenAIChatCompletionsHandler {
     size_t processedTokens = 0;  // tracks overall number of tokens processed by the pipeline
 
     absl::Status parseCompletionsPart();
-    absl::Status parseChatCompletionsPart();
+    absl::Status parseChatCompletionsPart(uint32_t maxTokensLimit);
```

Comment:
is it related to the issues from description?

Reply:
yes, it's related to adding support of max_completion_tokens